test/integration/targets/cron/tasks/main.yml

```diff
@@ -100,6 +100,31 @@
 - assert:
     that: remove_cron_file is not changed
 
+- name: Non regression test - cron file should not be empty after adding var (#71207)
+  block:
+  - name: Cron file creation
+    cron:
+      cron_file: cron_filename
+      name: "simple cron job"
+      job: 'echo "_o/"'
+      user: root
+
+  - name: Add var to the cron file
+    cron:
+      cron_file: cron_filename
+      env: yes
+      name: FOO
+      value: bar
+      user: root
+
+  - name: "Ensure cron_file still contains job string"
+    replace:
+      path: /etc/cron.d/cron_filename
```

Comment:
This path doesn't exist on Alpine, so this is breaking tests. Can make a block for these tests (similar to the one right below) that does `when: ansible_distribution != 'Alpine'` so they get skipped on Alpine?

Reply:
done ;)